pkg/kubelet/kubelet.go

```diff
@@ -1350,12 +1350,20 @@ func (kl *Kubelet) cleanupOrphanedPodDirs(pods []*api.Pod, runningPods []*kubeco
 		return err
 	}
 	errlist := []error{}
-	for i := range found {
-		if !active.Has(string(found[i])) {
-			glog.V(3).Infof("Orphaned pod %q found, removing", found[i])
-			if err := os.RemoveAll(kl.getPodDir(found[i])); err != nil {
-				errlist = append(errlist, err)
-			}
+	for _, uid := range found {
+		if active.Has(string(uid)) {
+			continue
+		}
+		if err := os.Remove(kl.getPodVolumesDir(uid)); err != nil {
```

Comment:
Agreed, not holding up this pr. 

What i meant was if you unmount a network device the directory will look empty wheres the rmdir syscall that os.remove invokes can returns all sorts of errors (http://man7.org/linux/man-pages/man2/rmdir.2.html).